Packs/Gmail/Integrations/Gmail/Gmail_test.py

```diff
@@ -128,6 +128,52 @@
 }
 
 
+EXPECTED_LIST_LABELS = {
```

Comment:
read the mock data from a file instead of having such a huge mock response as a variable

Reply:
I'm open to suggestions on how this should be done. I am just following the standard as per the other tests.

Reply:
not critical, I see in this integration it hasn't been used, so lets give up on that one that.

Reply:
Sure, NP.